src/services/recipe.service.ts

```diff
@@ -1,63 +1,66 @@
-import express from "express";
-import { Request, Response } from "express";
 import { Recipe, RecipeDocument } from "../models/recipe.model";
 
 type CreateRecipeBody = Omit<Recipe, "userId">;
 
-export async function createRecipe(recipe: Recipe): Promise<RecipeDocument> {
-  const newRecipe = new Recipe(recipe);
-  return await newRecipe.save();
-}
+export class RecipeService {
+  public async createRecipe(recipe: Recipe): Promise<RecipeDocument> {
+    const newRecipe = new Recipe(recipe);
```

Comment:
I think I'd skip assigning `newRecipe` and just do everything in one line - `return await new Recipe(recipe).save()`